recipes/sprit/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sprit" %}
+{% set version = "0.1.40" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sprit-{{ version }}.tar.gz
+  sha256: 5e710ad11e98fa0cafcff7d7a4b958adeacb1e27ef53d9f634c29dfdb7e38983
+
+build:
+  entry_points:
+    - sprit = sprit.sprit_cli:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.9
+    - obspy
+    - scipy
+    - matplotlib-base
+    - pandas
+    - numpy
+    - pyqt
```

Comment:
pyqt is quite a heavy dependency to add for something like a demo file


is there somewhere other than in https://github.com/spirit-code/spirit/blob/e82250d3b14411c2c2fa292d143f13e3e111ad8c/VFRendering/demo.py#L5 where it is being used?